scss/mixins/_type-mixins.scss

```diff
@@ -80,3 +80,7 @@
   font-weight: bold;
   padding: 0 0.3em;
 }
+
+@mixin t-upper {
```

Comment:
👍  all good. It's a fair question about `@extend` / `@include` . I don't have super strong thoughts on the matter other than that I know people tend to frown on `@extend`. But certainly not opposed outright.